admin_guide/scheduling/scheduler.adoc

```diff
@@ -97,6 +101,105 @@ and a string match with the name of the host.
 {"name" : "HostName"}
 ----
 
+**_NoVolumeZoneConflict_** checks that the volumes a pod requests 
+are available on the zone.
+----
+{"name" : "NoVolumeZoneConflict"}
+----
+
+**_MaxEBSVolumeCount_** Checks the maximum volume that can be attached to an AWS instance.
+----
+{"name" : "MaxEBSVolumeCount"}
+----
+
+**_MaxGCEPDVolumeCount_** Checks the maximum PD volume limit for GCE.
+----
+{"name" : "MaxGCEPDVolumeCount"}
+----
+
+**_MatchInterPodAffinity_** Checks if the pod affinity/antiaffinity rules permit the pod.
+----
+{"name" : "MatchInterPodAffinity"}
+----
+
+**_NoDiskConflict`_** Checks the volume requested by a pod is available.
+----
+{"name" : "NoDiskConflict"}
+----
+
+**_PodToleratesNodeTaints_** Checks if a pod can tolerate the node taints.
+----
+{"name" : "PodToleratesNodeTaints"}
+----
+
+**_CheckNodeMemoryPressure_** Checks if a pod can be scheduled on a node with a memory pressure condition.
+----
+{"name" : "CheckNodeMemoryPressure"}
+----
+
+**_CheckNodeDiskPressure_** Checks if a pod can be scheduled on a node with a disk pressure condition.
+----
+{"name" : "CheckNodeDiskPressure"}
+----
+
+**_NoVolumeNodeConflict_** 
+----
+{"name" : "NoVolumeNodeConflict"}
+----
+
+The following General Predicates check whether noncritical predicates and essential predicates pass. Noncritical Predicates are the predicates
+that only non-critical pods need and essential predicates are the predicates that all pods need.
+
+**_CheckVolumeBinding_**
+----
+{"name" : "CheckVolumeBinding"}
+----
+
+**_CheckNodeCondition_** Checks if a pod can be scheduled on a node reporting out of disk, network unavailable, and not ready condition.
+----
+{"name" : "CheckNodeCondition"}
+----
+
+**_HostName_**
+----
+{"name" : "HostName"}
+----
+
+**_PodFitsHostPorts_** Checks if a node has free ports for the requested pod ports.
```

Comment:
Seems like you have described it twice and there are others too similarly. please fix.